### PR TITLE
Audit all mocks to fit pattern

### PR DESCRIPTION
**Pull request by sourabhmarathe**

### CODE REVIEW

File: test/mocks/Element.sol [hunk 1]
```diff
@@ -23,33 +23,24 @@ struct FundManagement {
...
-    address fundManagementSenderCalled;
-    uint256 singleSwapAmountCalled;
-
-    function deadline() external view returns (uint256) {
-        return deadlineCalled;
+contract Element {
+    struct SwapArgs {
+        address recipient;
+        uint256 swapAmount;
+        uint256 _return;
```
Comment (sourabhmarathe) [inline test/mocks/Element.sol]: Look into this

Comment (sourabhmarathe) [inline test/mocks/Element.sol]: Look into this

File: pkg/lendertesting/lend_test.go [hunk 2]
```diff
@@ -372,7 +372,7 @@ func (s *lendTestSuite) TestLendElement() {
 	maturity := big.NewInt(100000)
 	elementPoolId := [32]byte{1}
 	amount := big.NewInt(10000)
-	returnAmount := big.NewInt(100)
```
Comment (robrobbins) [inline pkg/lendertesting/lend_test.go:375]: returnAmount is ok -- that wasn't part of the `return_` joke from Element (or whomever's) garbage code

Comment (robrobbins) [reply pkg/lendertesting/lend_test.go:375]: i mean that came from jule's `amount returned` comments... no idea what it actually should be called. maybe it _is_ actually `limit` -- think we are using `r` in the args...

Comment (robrobbins) [reply pkg/lendertesting/lend_test.go:375]: you can always change those as you learn more

**Review by robrobbins — APPROVED** [review]

**Pull request merged**